src/Illuminate/Database/Eloquent/Relations/Concerns/CanBeOneOfMany.php

```diff
@@ -214,7 +214,7 @@ protected function newOneOfManySubQuery($groupBy, $columns = null, $aggregate =
             }
         }
 
-        $this->addOneOfManySubQueryConstraints($subQuery, $groupBy, $columns, $aggregate);
+        $this->addOneOfManySubQueryConstraints($subQuery, column: null, aggregate: $aggregate);
```

Comment:
This was a tricky one:

The method only accepts 3 arguments while 4 were being passed in:
https://github.com/laravel/framework/blob/032f378ca43ea13249a08a53e481278d98bf4cb4/src/Illuminate/Database/Eloquent/Relations/Concerns/CanBeOneOfMany.php#L34-L42


However, the only two implementations don't even use the last two arguments:
https://github.com/laravel/framework/blob/032f378ca43ea13249a08a53e481278d98bf4cb4/src/Illuminate/Database/Eloquent/Relations/HasOne.php#L88-L91

https://github.com/laravel/framework/blob/032f378ca43ea13249a08a53e481278d98bf4cb4/src/Illuminate/Database/Eloquent/Relations/MorphOne.php#L86-L89

Additionally, the `$groupBy` and `$columns` variables that were being passed could be arrays whereas the `$column` parameter can only be `string|null`---given all this I just decided to pass `null` here for the `column` parameter.